tests/Contents.m

```diff
@@ -0,0 +1,40 @@
+% TESTS Unit tests for Rigbox.
+% Version xxx 02-Oct-2019
+%  The tests folder contains all tests and fixtures for testing core Rigbox
```

Comment:
I would remove the 'quickly' in this paragraph, as this is subjective. We should also really separate out MC and SC tests